synapse/lib/storm.py

```diff
@@ -2762,7 +2777,26 @@ def _print_optarg(self, names, argdef):
             cstr = ', '.join(str(c) for c in choices)
             helpstr = f'{helpstr} (choices: {cstr})'
 
-        self._printf(f'{base}: {helpstr}')
+        helplst = helpstr.split('\n')
+        if helplst and not helplst[0].strip():
+            helplst = helplst[1:]
+        min_space = min((len(ln) - len(ln.lstrip()) for ln in helplst if ln.strip()), default=0)
+
+        base_w = 32
```

Comment:
Can we figure out the minimum amount of space required for all arguments of this command instead of hardcoding it to 32?

Reply:
I'd say this is outside of scope since the default was 32 before this pr.